synthtool/gcp/templates/node_library/.github/ISSUE_TEMPLATE/documentation_request.yaml

```diff
@@ -0,0 +1,53 @@
+name: Documentation Requests
+description: Requests for more information
+body:
```

Comment:
just checking because the other templates are in markdown, should this be a markdown file also?

Reply:
Looks like instead the others were changed to yaml :)